src/components/Navbar/Navbar.jsx

```diff
@@ -1,30 +1,69 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import { Button } from '../';
+import './Navbar.scss';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars, faTimes } from '@fortawesome/free-solid-svg-icons';
+import pibpLogo from '../../images/pibpLogo.png';
 
 export function Navbar() {
+    const [navVisibility, setNavVisibility] = useState('');
+
+    let changeNavVisibility = () => {
+        let visibility = navVisibility === '' ? 'navbar__visible' : '';
+        setNavVisibility(visibility);
+    };
+
     return (
-        <nav>
-            <h1>LOGO</h1>
-            <ul>
+        <nav className="navbar">
+            <img className="navbar__logo" src={pibpLogo} alt="PiBP Logo" />
+
+            <ul className={`navbar__list ${navVisibility}`}>
```

Comment:
and then here use template string to use proper className ${navVisibility ? 'navbar__visible' : ''}